src/dtoannotations/AdulthoodValidationProcess.java

```diff
@@ -0,0 +1,29 @@
+package dtoannotations;
+
+import java.lang.reflect.Field;
+import java.time.LocalDate;
+
+public class AdulthoodValidationProcess {
+
+    public String ValidateAdulthoodAnnotation (Object object) throws IllegalAccessException {
+        String text = "";
+        Field[] declaredFields = object.getClass().getDeclaredFields();
+        for (Field declaredField : declaredFields) {
+            declaredField.setAccessible(true);
+            Object fieldValue = declaredField.get(object);
```

Comment:
**cast** this field into LocalDate and also, you can remove throws from method signature and catch it here.